recipes/jpcre2/all/conanfile.py

```diff
@@ -1,33 +1,44 @@
-from conans import ConanFile, tools
 import os
 
-required_conan_version = ">=1.33.0"
+from conan import ConanFile
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.52.0"
 
 
 class Jpcre2Conan(ConanFile):
     name = "jpcre2"
-    homepage = "https://github.com/jpcre2/jpcre2"
     description = "Header-only C++ wrapper for PCRE2 library."
-    topics = ("regex", "pcre2", "header-only", "single-header")
-    url = "https://github.com/conan-io/conan-center-index"
     license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jpcre2/jpcre2"
+    topics = ("regex", "pcre2", "header-only", "single-header")
 
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def requirements(self):
         self.requires("pcre2/10.37")
```

Comment:
The readme states that a bump should be ok https://github.com/jpcre2/jpcre2#dependency
```suggestion
        self.requires("pcre2/10.42")
```